test/new_relic/agent/distributed_tracing/trace_context_cross_agent_test.rb

```diff
@@ -253,7 +253,11 @@ def verify_attributes(test_case_attributes, actual_attributes, event_type)
               %Q(Unexpected #{event_type} attribute "#{key}")
           end
 
-          # TODO: check vendors in test_case_attributes
+          (test_case_attributes['vendors'] || []).each do |k, v|
+            assert_equal v,
+              actual_attributes[k.to_s],
+              %Q(Wrong "#{k}" #{event_type} attribute; expected #{v.inspect}, was #{actual_attributes[k.to_s].inspect})
```

Comment:
It is my understanding that `k` and `v` are always strings for this new test. Is that correct? If so, all of the `to_s` and `#inspect` calls can be removed. Also NOTE that by virtue of using interpolation (`#{}`), Ruby will automatically call `#to_s` on the interpolated object.

Reply:
addressed with https://github.com/newrelic/newrelic-ruby-agent/pull/1692